content/docs/user-guide/how-to/add-output-to-stage.md

```diff
@@ -0,0 +1,69 @@
+# Add Output to Stage
+
+There are situations where you might have forgotten to specify one of the
+<abbr>outputs</abbr> for a stage while creating it, and then later want to add
+an output to that stage.
```

Comment:
Correct but this is not the only case where this scenario happens. The more generalized explanation here I think has to do with having run a stage without specifying all the outpus in dvc.yaml (either via run or manually) and later wanting to make some of the files/dirs produces by the state as outputs formally.

Reply:
(Without rerunning the stage)

Reply:
So pleas give the more general explanation and/or list all the ways that you can think about that a user would end up with this problem.